ts/src/header-validator/validate-os.ts

```diff
@@ -22,15 +22,15 @@ function validateURL(ctx: Context, member: InnerList | Item): void {
       (url.hostname === 'localhost' || url.hostname === '127.0.0.1')
     )
   ) {
-    ctx.error('must contain a potentially trustworthy URL')
+    ctx.warning('ignored, must contain a potentially trustworthy URL')
```

Comment:
This isn't part of the spec, I'm planning on removing this check/warning in a follow up PR.

Reply:
Follow-up PR: https://github.com/WICG/attribution-reporting-api/pull/1198